tests/test_mkvenv.zunit

```diff
@@ -54,7 +54,7 @@
     run cat "$TARGET/myproject/.venv"
 
     assert $status equals 0
-    assert "$output" matches "myproject-[a-zA-Z0-9]{8}"
+    assert "$output" matches "myproject-[a-z]{4}"
```

Comment:
I think I'd prefer to keep this as 8 characters if thats ok. Cant think of a string reason to move away from this

Reply:
I made it shorter because the venv name shows up in the prompt and eats up my terminal width with random noise. 4 chars gets us 1 in half a million, which felt unlikely enough to me. You'd need like 800 duplicate venvs to reasonably expect a collision.

But if you comment here again, I'll change it :-)

Reply:
Hard to argue with that logic :) I was mostly referring to consistency between releases but I dont think it would affect users too much anyway.

Feel free to keep this here then